src/tests/Common/XUnitLogChecker/XUnitLogChecker.cs

```diff
@@ -489,15 +489,14 @@ static void PrintStackTracesFromDumps(string testLogPath)
             );
 
             dumpsFound = Directory
-                         .GetFiles(s_configuration.DumpsPath, "*coredump*.dmp")
```

Comment:
@ivdiazsa some minor changes to XUnitLogChecker.

The libraries tests generate dump files that begin with either `dotnet` or `dotnet.exe` (depending on the OS) unlike the coreclr ones which get generated with the keyword coredump in their names.

Reply:
That's fine. The name change in the `if` is unnecessary because that section is only triggered in coreclr tests, but it causes no harm, so feel free to keep it uniform if you want.